src/AmalgamatorSession.ts

```diff
@@ -451,6 +456,24 @@ export class AmalgamatorSession extends LoggingDebugSession {
         }
     }
 
+    protected async disassembleRequest(
+        response: DebugProtocol.DisassembleResponse,
+        args: DebugProtocol.DisassembleArguments
+    ): Promise<void> {
+        response.body = {
+            instructions: [],
+        };
+        if (args.memoryReference) {
+            const [childDap, memoryReference] = this.disassembleHandles.get(
+                parseInt(args.memoryReference)
```

Comment:
Rather than parseInt here, extract the index from the `args.memoryReference` and use that to find the correct `childDap` in `this.childDaps`. Then strip off the index (and separator) and set `args.memoryReference` to that value to pass to `childDap.disassembleRequest`

Reply:
I have updated it. If it is not your idea, please help me clear your idea.